<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>source_widgets</class>
 <widget class="QFrame" name="source_widgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>126</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>126</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="label_media_type">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Lucida Grande</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_media_path">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lucida Grande</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
          </font>
         </property>
         <property name="placeholderText">
          <string>Media Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="m_hovered_pushbutton_round_close" name="pushButton_close">
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPuhButton:flat{border: none;}
QPuhButton{background:transparent;}
QPushButton:checked{ background-color: white; border: none;  }
QPushButton:!checked{ background-color: white; border: none;  } 
QPushButton:clicked{ background-color: white;  border: none;}               
QPushButton:pressed { background-color: white; border: none; }




</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_repoint">
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPuhButton:flat{border: none;}
QPuhButton{background:transparent;}
QPushButton:checked{ background-color: white; border: none;  }
QPushButton:!checked{ background-color: white; border: none;  } 
QPushButton:clicked{ background-color: white;  border: none;}               
QPushButton:pressed { background-color: white; border: none; }




</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_media_path_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lucida Grande</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
          </font>
         </property>
         <property name="placeholderText">
          <string>Media Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="m_hovered_pushButton" name="pushButton_close_2">
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPuhButton:flat{border: none; }
QPuhButton{background:transparent;}
QPushButton:checked{ background-color: white; border: none;  }
QPushButton:!checked{ background-color: white; border: none;  } 
QPushButton:clicked{ background-color: white;  border: none;}               
QPushButton:pressed { background-color: white; border: none; }
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_repoint_2">
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPuhButton:flat{border: none;}
QPuhButton{background:transparent;}
QPushButton:checked{ background-color: white; border: none;  }
QPushButton:!checked{ background-color: white; border: none;  } 
QPushButton:clicked{ background-color: white;  border: none;}               
QPushButton:pressed { background-color: white; border: none; }




</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>m_hovered_pushButton</class>
   <extends>QPushButton</extends>
   <header location="global">ui_custom_classes/m_hovered_pushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>m_hovered_pushbutton_round_close</class>
   <extends>QPushButton</extends>
   <header location="global">ui_custom_classes/m_hovered_pushbutton_round_close.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
